spec/furniture/marketplace/stripe_events_controller_request_spec.rb

```diff
@@ -22,18 +26,30 @@
     double(Stripe::Charge, balance_transaction: "btx_2234")
   }
 
+  let(:square_client) {
+    double(Square::Client)
+  }
+
+  let(:square_orders_api) {
+    double(Square::OrdersApi)
+  }
+
   before do
     allow(Stripe::Webhook).to receive(:construct_event).with(anything, "sig_1234", marketplace.stripe_webhook_endpoint_secret).and_return(stripe_event)
 
     allow(Stripe::PaymentIntent).to receive(:retrieve).with("pi_1234", anything).and_return(payment_intent)
     allow(Stripe::Transfer).to receive(:create).and_return(double(Stripe::Transfer, id: "st_fake_1234"))
     allow(Stripe::BalanceTransaction).to receive(:retrieve).with("btx_2234", anything).and_return(balance_transaction)
     allow(Stripe::Charge).to receive(:retrieve).with("ch_1234", anything).and_return(charge)
+    allow(Square::Client).to receive(:new).with(access_token: anything, environment: anything).and_return(square_client)
```

Comment:
I prefer to use more specific values than `anything` so that tests will fail if we start passing in nonsense (i.e. missing an access token or environment)

Reply:
Yeah agree. I believe I may have used this as a placeholder. Good catch.